index.html

```diff
@@ -5,9 +5,57 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Document</title>
+    <title>Portfolio Project</title>
 </head>
 <body>
-    <h1 class="header">Hello</h1>
+    <header>
+        <nav>
+            <img src="./images/menu.svg" alt="menu">
+        </nav>
+    </header>
+<section class="headline">
+        <div class="headline-header">
+		<h1>
+			Hey there.<br/>I'm Demes.<br/>
```

Comment:
- [optional] please kindly add paddings and line spacing to the different elements of your headline, so they can follow the Figma version. Also ensure the social icons go down a little like in the original version. Like the previous reviewer noted, paying attention to details matters to the recruiters. Keep up the spirit 😄